packages/nimbus/src/components/date-range-picker/date-range-picker.dev.mdx

```diff
@@ -0,0 +1,413 @@
+---
+title: DateRangePicker Component
+tab-title: Implementation
+tab-order: 3
+---
+
+# DateRangePicker
+
+The DateRangePicker component enables users to select a date range through two flexible input methods: typing dates directly into input fields or selecting from a visual calendar. It's ideal for filtering data, scheduling events, and any scenario requiring start and end date selection.
+
+## Getting started
+
+### Import
+
+```jsx
+import { DateRangePicker } from '@commercetools/nimbus';
+```
+
+For TypeScript projects:
+```tsx
+import { DateRangePicker, type DateRangePickerProps } from '@commercetools/nimbus';
+```
```

Comment:
nit to think about. maybe we can tighten this part up a bit by doing:

```tsx
// Basic import
import { DateRangePicker } from '@commercetools/nimbus';

// TypeScript
import { DateRangePicker, type DateRangePickerProps } from '@commercetools/nimbus';
```

<img width="531" height="248" alt="Screenshot 2025-11-04 at 14 54 57" src="https://github.com/user-attachments/assets/184eaa20-587b-4fa3-bad2-7a435696d640" />